### DISCUSSION TITLE
Meeting of Remix CC (2023-08-27)

### DISCUSSION THREAD

**Discussion by drafish** [PlanckerLabs/remix-cc#58] [Meeting of Remix CC]
### Remix 团队是怎么招人的
关于这个问题，我去年就有一个猜想，这周刚刚得到了印证，趁着这次周会给大家分享下。

这个猜想不光是针对 Remix 的，而是针对所有开源项目的。

因为开源项目是开放开源的，所有人都可以来参与，所以他们没有必要走传统公司那样的招聘流程，完全可以在贡献者中招合适的人。这样做有两个好处

1. 面试成本很低，因为面试之前他们就已经对贡献者的技术水平有了一个基本了解
2. 磨合期很短，因为新人在正式加入团队之前已经跟团队合作过一段时间

我去年就看到 Remix 团队在 remoteok 上发布的岗位，那会儿我就想应聘，但对自己的英文水平没什么信心，所以没敢去应聘。不过我还是对他们的招聘模式就做了一番猜想。

这周我向 Rob 旁敲侧击地打听了一下，他们去年招的人是谁。当 Rob 跟我说他们招的人是 Joe 时，我就知道我猜的没错。因为我看过 Joe 的 PR 记录，最早的 PR 是2021年8月。所以说他在正式加入 Remix 团队之前已经至少和 Remix 团队合作了一年。甚至我印象中 Joe 的名字很早就出现在 Remix 官网的团队介绍中了，应该是在 Remix 团队发布招聘信息之前就已经在了的。

所以说当 Remix 团队发布招聘信息的时候，其实已经差不多内定了。招聘只是走一个流程，并且向社区传达一个信息，就是说他们在招人。这个说起来有点像萝卜坑，但我并不反感这种形式，我反而觉得开源项目就是应该这样招人。

另外，我还想说的一点就是，我之所以那么想加入 Remix 团队，是因为 Remix 是以太坊的核心项目，名声摆在那里。但这也说明了另一个问题，就是中文区在公共物品建设方面长期出于空白的状态，不管是氛围还是资金，较之于国际社区都是明显的不足。如果中文区有良好的氛围，有足够的资金，那完全可以自己诞生出优秀的公共物品。那我们中文区的人就没必要舍近求远，在自家社区里面就能找到合适的项目参与进去。而现在普朗克和GCC在做的事情，就是在填补这个空白。

**Comment by drafish** [comment]
### DJQ  (2023-08-20)
1. 完成了 Remix IDE 国际化开发的收尾工作，已经提交 PR 给官方了 -- https://github.com/ethereum/remix-project/pull/3992
2. 以 Remix IDE 国际化为主题写了一篇文章 -- https://github.com/PlanckerLabs/remix-cc/discussions/60
3. 和 Remix 官方开了同步会议，反馈了一些问题，并且提了一些建议 -- https://github.com/PlanckerLabs/remix-cc/discussions/56
4. 之前翻译的 LearnETH 教程收到了一些修改意见，建了 issue ，并且 assign 给了 mocca -- https://github.com/PlanckerLabs/remix-cc/issues/55
5. 国内镜像站更新到了 v0.35.0 版本，并且翻译了 v0.35.0 版本的 release notes -- https://medium.com/@remix-cc/b4120ed5d8ec
6. 与 Footprint Analytics 的 CEO 开了个会，简单聊了下双方在做的事儿，后续可能会有合作 -- https://www.footprint.network/ 

### DJQ（2023-08-27）
1. 在 PET 上做了一次技术分享，主题是 Remix IDE 国际化
2. 帮 Moxi review 一下他的 PR ，并且在 Remix 同步会上帮他推进了一下
3. 在 Remix 那边提了一个提案，建议他们使用 github discussions 做为官方论坛
4. 发现了 Remix 代码中 prettier 和 eslint 规则存在冲突，已反馈给官方
5. Remix 官方对 codebase 进行了全面的 formate ，导致我之前提交的国际化 PR 出现大面积冲突，所以 rebase 了一下并且解决了冲突

**Comment by gongzishuye** [comment]
mocca（双周）
1. LearnETH教程的翻译有很多需要校正的地方，完成了校正并合并到了zh分支 #55
